File: level89/homework/level89.js
// davaleba 2
// მომხმარებლისგან ამოჭრის საწყისი და ბოლო ინდექსები
let userString = prompt("შეიტანეთ სტრინგი:");
let startIndex = parseInt(prompt("შეიტანეთ საწყისი ინდექსი:"));
let endIndex = parseInt(prompt("შეიტანეთ ბოლო ინდექსი:"));

// ამოჭრა სტრინგის
let slicedString = userString.slice(startIndex, endIndex);

// შედეგის გამოჩენა
console.log("ამოჭრილი სტრინგი:", slicedString);






// davaleba 5
function reverseWords(sentence) {
    // წინადადების გაყოფა სიტყვებად
    let words = sentence.split(' ');

    // თითოეული სიტყვა შემოტრიალება
    let reversedWords = words.map(word => word.split('').reverse().join(''));

    // სიტყვების დაბრუნება ერთიან წინადადებად
    return reversedWords.join(' ');
}

// გამოსაყენებელი მაგალითი
let sentence = "amis kodi damiwere";
let result = reverseWords(sentence);
console.log(result);

File: level89/homework/level89.py
# davaleba 1
# მომხმარებლისგან სტრინგი და ინდექსების მიღება
user_string = input("შეიტანეთ სტრინგი: ")
start_index = int(input("შეიტანეთ დასაწყისი ინდექსი: "))
end_index = int(input("შეიტანეთ დასასრული ინდექსი: "))

# slicing-ის გამოყენებით ამოჭრა სტრინგის
sliced_string = user_string[start_index:end_index]

print("ამოჭრილი სტრინგი:", sliced_string)




# davaleba 4
def reverse_words(sentence):
    # სიტყვების გაყოფა
    words = sentence.split()
    # თითოეული სიტყვის შემოტრიალება და მათ სიაში დაბრუნება
    reversed_words = [word[::-1] for word in words]
    # სიტყვების გაერთიანება მათ ახალ რიგში
    return ' '.join(reversed_words)

# შეტყობინების მიღება
input_sentence = input("მიუთითეთ წინადადება: ")
print(reverse_words(input_sentence))
